debian/kano-touch-support.install

```diff
@@ -1,3 +1,5 @@
-touch-detect/touch-detect usr/bin/
-touch-detect/*.sh  usr/share/kano-peripherals/scripts/
-conf/xorg.conf.d/* usr/share/kano-peripherals/xorg.conf.d/
+touch-detect/release/bin/touch-detect usr/bin/
+touch-detect/release/lib/*.so usr/lib/
+touch-detect/scripts/*.sh  usr/share/kano-peripherals/scripts/
+touch-detect/conf/xorg.conf.d/* usr/share/kano-peripherals/xorg.conf.d/
+touch-detect/etc/* etc/
```

Comment:
It'd make more sense if the contents of this folder was under `touch-detect/conf/`

Reply:
Yeah, you're probably right. It is just because this stuff all gets installed exactly as-is into `/etc` while the stuff in `touch-detect/conf` gets dumped into `usr/share/kano-peripherals` (moved from the root `conf` directory as-is so that we don't need to do selective install. Maybe I'll just put the `touch-detect/etc/` as a subdirectory of `touch-detect/conf/` or maybe I'll move the `conf` stuff out because if it gets put in `/usr/share` then it probably should be a resource or something else.

Reply:
I think at a project level the structure is quite important. My suggestion is to use `conf/` as a bucket for everything configuration. How you organise it lower than that probably varies on what you have, where it's installed, etc